crates/freya/src/lib.rs

```diff
@@ -8,15 +8,16 @@
 //!
 //! Powered by [🧬 Dioxus](https://dioxuslabs.com) and [🎨 Skia](https://skia.org/).
 //!
-//! - [`Elements`](freya_elements::elements#structs)
-//! - [`Events`](freya_elements::elements#functions)
-//! - [`Components`](freya_components)
-//! - [`Hooks`](freya_hooks)
-//! - [`Theming`](self::_docs::theming)
-//! - [`Hot reload`](self::_docs::hot_reload)
-//! - [`Testing`](self::_docs::testing)
-//! - [`Animating`](self::_docs::animating)
-//! - [`Devtools`](self::_docs::devtools)
+//! - [Elements API reference](freya_elements::elements#structs)
+//! - [Events API reference](freya_elements::elements#functions)
+//! - [Elements and events guides](freya_elements::_docs)
```

Comment:
What about naming this to:
```suggestion
//! - [Style and layout explanations](freya_elements::_docs)
```

Reply:
or "references" instead of "explanations"

Reply:
> "Style and layout" is misleading because the inheritance "guide" is neither of those. I know there's nothing there about events, but I believe there will be in the future. 

What about `Supported colors, layout and inherited values`, also, if there are no events, let's not call it events, this are versioned docs, if there is ever some events explanation this can be changed.